app.rb

```diff
@@ -1,7 +1,30 @@
 require 'sinatra/base'
+require_relative './lib/player.rb'
+require_relative './lib/game.rb'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  set :public_folder, 'public'
+  get '/' do
+    erb :index
+  end
+
+  post '/names' do
+    @game = Game.create(Player.new(params[:name]))
```

Comment:
Cool that you managed to do this all in one line, but for readability consider splitting this up into multiple lines.